tests/base_tests.py

```diff
@@ -15,5 +16,8 @@ def tearDown(self):
     def generate_random_text():
         return j.data.idgenerator.chars(10)
 
+    def random_name(self):
+        return j.data.idgenerator.nfromchoices(10, string.ascii_letters)
```

Comment:
Can be replaced by

```suggestion
        return j.data.idgenerator.chars(10)
```

Reply:
some names invalid like name start with number

![image](https://user-images.githubusercontent.com/31102305/97999918-94c38780-1df4-11eb-9d4e-b5838bcf2dc9.png)